### PR TITLE
Related clang stuff removed

### PR DESCRIPTION
**Pull request by lalitmee**
This removes the clang related code from ExceptionsTest.py and
ConsoleInteractionTest.py. clanglib-py3 is changed to version_1 and
version_2 and ClangCloneDetectionBear is changed to CloneDetectionBear.

Closes https://github.com/coala/coala/issues/5250

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Please dont use `stuff`

**Comment by lalitmee** [comment]
okk. If I use `code`, will it be right?

**Comment by lalitmee** [comment]
@li-boxuan , thank you for pointing out. I didn't see that. Now I have removed that `stuff` thing. :innocent:

File: tests/output/ConsoleInteractionTest.py [hunk 1]
```diff
@@ -755,7 +755,7 @@ def test_print_results_multiple_ranges(self):
             print_results(
                 self.log_printer,
                 Section(''),
-                [Result('ClangCloneDetectionBear',
+                [Result('CloneDetectionBear',
```
Comment (jayvdb) [inline tests/output/ConsoleInteractionTest.py]: dont use semi-real values.  that is confusing.

make it obviously fake, so the reader knows it is fake.

File: tests/misc/ExceptionsTest.py [hunk 2]
```diff
@@ -11,6 +11,6 @@ def test_get_exitcode(self):
         self.assertEqual(get_exitcode(AssertionError()), 255)
         self.assertEqual(get_exitcode(SystemExit(999)), 999)
         self.assertEqual(get_exitcode(VersionConflict(
-            'libclang-py3 0.3', 'libclang-py3==0.2')), 13)
+            'version_1', 'version_2')), 13)
```
Comment (jayvdb) [inline tests/misc/ExceptionsTest.py]: these are not valid version strings.

Comment (lalitmee) [reply tests/misc/ExceptionsTest.py]: Actually it means that there are two different versions, that's why I put `version_1` and `version_2`.

File: tests/output/ConsoleInteractionTest.py (hunk 1 above)
Comment (lalitmee) [reply tests/output/ConsoleInteractionTest.py]: okk

**Comment by jayvdb** [comment]
Ack ExceptionsTest.py: Clang code removed

**Comment by jayvdb** [comment]
Ack https://github.com/coala/coala/commit/1f093c6025cc8c24d79dd955de159fd1b307fe6b

**Comment by jayvdb** [comment]
Ack 1f093c6

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**